__tests__/__system__/cli/define/urimap-pipeline/cli.define.urimap.pipeline.system.test.ts

```diff
@@ -70,12 +70,22 @@ describe("CICS define urimap-pipeline command", () => {
         const urimapNameSuffixLength = 4;
         const urimapName = "DFN" + generateRandomAlphaNumericString(urimapNameSuffixLength);
         const options: IURIMapParms = { name: urimapName, csdGroup, regionName };
-        const output = runCliScript(__dirname + "/__scripts__/define_urimap_pipeline.sh", TEST_ENVIRONMENT,
+        let output = runCliScript(__dirname + "/__scripts__/define_urimap_pipeline.sh", TEST_ENVIRONMENT,
             [urimapName, csdGroup, "urimap/test/invalid.html", "www.example.com", "FAKEPIPE", regionName]);
-        const stderr = output.stderr.toString();
+        let stderr = output.stderr.toString();
         expect(stderr).toEqual("");
         expect(output.status).toEqual(0);
         expect(output.stdout.toString()).toContain("success");
+
+        output = runCliScript(__dirname + "/__scripts__/get_resource_urimap.sh", TEST_ENVIRONMENT,
+            [regionName,
+                csdGroup]);
+        stderr = output.stderr.toString();
+        expect(stderr).toEqual("");
```

Comment:
Thanks for placing the `stderr == ""` check before checking the output status. that way we get to see the error message (in case there is any) when running the tests instead of just `assert(1==0)`